report/sql_tasks/tasks/report/create_from_scratch/03_public/01_materialized_views/03_lms_events/01_create_view.sql

```diff
@@ -0,0 +1,27 @@
+DROP MATERIALIZED VIEW IF EXISTS lms_events CASCADE;
+
+CREATE MATERIALIZED VIEW lms_events AS (
```

Comment:
If we ever are interested in other event types we might find out that materealizing those alone in a separate view makes sense as for speed this table is 99% lunches.